packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.service.js

```diff
@@ -91,6 +92,13 @@ export default class IpAgoraOrder {
     );
   }
 
+  checkIpDedicatedServerIsOrderable(serviceName) {
+    return this.$http
+      .get(`/dedicated/server/${serviceName}/orderable/ip`)
+      .then(({ data: orderable }) => !!orderable.ipv4?.length)
```

Comment:
This would be more explicit
```suggestion
      .then(({ data: orderable }) => orderable.ipv4?.lenght > 0)
```